src/WordPress/Blueprints/Runner/Step/UnzipStepRunner.php

```diff
@@ -2,21 +2,33 @@
 
 namespace WordPress\Blueprints\Runner\Step;
 
+use WordPress\Blueprints\BlueprintException;
 use WordPress\Blueprints\Model\DataClass\UnzipStep;
 use WordPress\Blueprints\Progress\Tracker;
 use function WordPress\Zip\zip_extract_to;
 
 class UnzipStepRunner extends BaseStepRunner {
 
+	/**
+	 * Runs the Unzip Step
+	 *
+	 * @param UnzipStep    $input Step.
+	 * @param Tracker|null $progress Tracker.
+	 * @return void
+	 */
 	public function run(
 		UnzipStep $input,
 		Tracker $progress = null
```

Comment:
Let's require the tracker 

```suggestion
		Tracker $progress
```

Reply:
Done.

Reply:
Done.